README.md

```diff
@@ -106,28 +106,36 @@ Support for other OpenTracing implementations can easily added here.
 
 #### Improving performance with an SSD
 
-To improve performance of dependency installation, the server can be configured to use a mounted SSD at a given directory
-
-Example for Kubernetes:
-
-1. Add a volume mount to the container spec:
-
-```diff
-  image: sourcegraph/lang-typescript
-+ volumeMounts:
-+   - mountPath: /mnt/cache
-+     name: cache-ssd
-```
-
-TODO `volumes` entry with `emptyDir`?
-
-2. Tell the language server to use the mount as the root for temporary directories:
-
-```diff
-  env:
-+   - name: CACHE_DIR
-+     value: /mnt/cache
-```
+To improve performance of dependency installation, the server can be configured to use a mounted SSD at a given directory by setting the `CACHE_DIR` environment variable. The instructions for how to mount a SSD depend on your deployment environment.
+
+##### Example for Kubernetes:
```

Comment:
Note that this header is a subheader of "Using Kubernetes". Do you think the document should be restructured?

Reply:
Yeah, I think it would be useful to have top level configuration doc that is deployment independent, and then in each section have an example for Kubernetes.